### ISSUE TITLE
Mixpanel: Matic message count is incorrect/static

### ISSUE THREAD

**Issue by verabehr** [VolumeFi/paloma#39]
The Matic message count in mixpanel is incorrect. It says 1k, but we have 2.2k. This is the contract: https://polygonscan.com/address/0x5dfb2a810f14236d91509c9144344a26aee729b1

**Comment by rootedbox** [comment]
I removed POLY_MESSAGE_COUNT in a prior ticket. should I add it back in? @verabehr

**Comment by verabehr** [comment]
Yes, please. I don't know why it was removed but that was an error
